src/jquery.multiselect.js

```diff
@@ -712,21 +894,50 @@
         }
       });
     },
+
     /**
      * Binds all events used in the widget
      * This calls the menu, button, and header event binding methods
      */
     _bindEvents: function() {
       var self = this;
 
-      self._bindButtonEvents();
+      if (!self.options.listbox) {
+         self._bindButtonEvents();
+      }
       self._bindMenuEvents();
       self._bindHeaderEvents();
 
-      // Close each widget when clicking on any other element/anywhere else on the page
-      //   or scrolling w/ the mouse wheel outside the menu button.
-      self.document.on('mousedown.' + self._namespaceID + ' wheel.' + self._namespaceID + ' mousewheel.' + self._namespaceID, function(event) {
-        if ( self._isOpen && !$(event.target).closest('.ui-multiselect,.ui-multiselect-menu').length ) {
+      // Set up resizable if the option is enabled and resizable is loaded.
+      if (!!self.options.resizableMenu && $.ui && 'resizable' in $.ui) {
+         self.$menu.show();
+         self.$menu.resizable({
+            containment: 'parent',
+            handles: 's',
+            helper: 'ui-multiselect-resize',
+            stop: function(e, ui) {
+               // Force consistent width
+               ui.size.width = ui.originalSize.width;
+               $(this).outerWidth(ui.originalSize.width);
+               if (self._trigger('resize', e, ui) !== false) {
+                  self.options.menuHeight = ui.size.height;
+               }
+               self._setMenuHeight(true);
+            }
+         });
+         self.$menu.hide();
+      }
+
+      // Close each widget when clicking on any other element/anywhere else on the page,
+      // another widget instance, or when scrolling w/ the mouse wheel outside the menu button.
+      self.document.on('mousedown.' + self._namespaceID
```

Comment:
These document listeners don't get destroyed when the widget is destroyed, and are created for each widget.

If the widget is destroyed and a new one created, these leak and multiply.

Reply:
Ok, I will have to address that in the destroy.

Reply:
Looking at the following below... doesn't `this.document.off(this._namespaceID);`  get rid of the document listeners for this widget's namespace?
```
    destroy: function() {
      // remove classes + data
      $.Widget.prototype.destroy.call(this);

      // unbind events
      this.document.off(this._namespaceID);
      $(this.element[0].form).off(this._namespaceID);

      if (!this.options.listbox) {
         this.$button.remove();
      }
      this.$menu.remove();
      this.element.show();

      return this;
    },

```

Reply:
That's what it is supposed to be doing, and in master it appears to be (I just double checked). I wonder if another change along the way broke that?

Reply:
It looks like I was removing the initial period from the namespace ID and it is needed for off().